go.mod

```diff
@@ -6,80 +6,85 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/DATA-DOG/go-txdb v0.1.3
 	github.com/DataDog/datadog-go v4.8.2+incompatible
-	github.com/aws/aws-sdk-go v1.34.28
+	github.com/aws/aws-sdk-go v1.44.327
 	github.com/getsentry/sentry-go v0.12.0
 	github.com/go-kit/kit v0.9.0
 	github.com/google/uuid v1.6.0
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0
 	github.com/magefile/mage v1.15.0
+	github.com/redis/go-redis/v9 v9.5.1
 	github.com/rs/cors v1.7.0
-	github.com/sirupsen/logrus v1.7.0
+	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.8.4
 	github.com/twmb/franz-go v1.12.1
 	github.com/twmb/franz-go/pkg/kmsg v1.4.0
 	google.golang.org/grpc v1.60.1
 	google.golang.org/protobuf v1.32.0
-	gopkg.in/DataDog/dd-trace-go.v1 v1.47.0
+	gopkg.in/DataDog/dd-trace-go.v1 v1.62.0
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
 	gorm.io/driver/mysql v1.4.6
 	gorm.io/driver/sqlite v1.4.4
-	gorm.io/gorm v1.24.5
+	gorm.io/gorm v1.25.3
```

Comment:
[q]: Did this happen due to `dd-trace-go` update? 🤔

Reply:
@laynax Yes, we have to upgrade `dd-trace-go` in order to have instrumentation for the `go-redis` library